src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/CompileItemHandler.cs

```diff
@@ -57,6 +61,23 @@ public void Handle(IComparable version, BuildOptions added, BuildOptions removed
             ApplyProjectBuild(version, difference, isActiveContext, logger);
         }
 
+        public void HandleProjectUpdate(IComparable version, IProjectChangeDescription projectChange, bool isActiveContext, IProjectLogger logger)
+        {
+            Requires.NotNull(version, nameof(version));
+            Requires.NotNull(projectChange, nameof(projectChange));
+            Requires.NotNull(logger, nameof(logger));
+
+            VerifyInitialized();
+
+            if (projectChange.Difference.RenamedItems.Any())
```

Comment:
Is the `.Any()` call helping here at all?

Reply:
I'll remove this

---
In reply to: [288348164](https://github.com/dotnet/project-system/pull/4876#discussion_r288348164) [](ancestors = 288348164)